docs/source/developers/python.rst

```diff
@@ -306,6 +299,10 @@ adding flags with ``ON``:
 Anything set to ``ON`` above can also be turned off. Note that some compression
 libraries are needed for Parquet support.
 
+For better understanding of the error in case the build fails add flag
+``-DCMAKE_BUILD_TYPE=debug``. Read more about build type in C++ building section
```

Comment:
I think this flag doesn't help while building, but only afterwards when running? (so the "in case the build fails" is then not correct; it's still useful to point to the different build types though)